lib/aws/bedrock.py

```
@@ -0,0 +1,36 @@
+            boto3.Session(profile_name=self.profile_name)
+            if self.profile_name
+            else boto3.Session()
+        )
+        self.client = session.client("bedrock-runtime", region_name=self.region_name)
+
+    def __getattr__(self, name: str) -> Any:
+        """Delegate Bedrock Runtime API access to the boto3 client."""
+        try:
+            return getattr(self.client, name)
+        except AttributeError:
+            raise AttributeError(f"'BedrockClient' object has no attribute '{name}'")
```

Ruff (0.15.11)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)